kafka-connect-s3/src/test/java/io/confluent/connect/s3/integration/S3SinkConnectorIT.java

```diff
@@ -723,6 +792,9 @@ private void setupProperties() {
     // converters
     props.put(KEY_CONVERTER_CLASS_CONFIG, JsonConverter.class.getName());
     props.put(VALUE_CONVERTER_CLASS_CONFIG, JsonConverter.class.getName());
+    props.put(CONTINUUM_BOOTSTRAP_SERVERS_CONFIG, connect.kafka().bootstrapServers());
```

Comment:
Just a thought: I wonder if it would be worth having any tests that don't enable the continuum stuff. Like if we left it disabled for all of the previously-existing tests, have the new test enable it, and another new test that verifies partition 0 by default? That could also help with the `verifyContinuumNotificationOffsets` thing.

Reply:
Sure, I did that. I still think it makes sense to have a boolean flag for verifying the Continuum messages in the test method that does the rest of the verification, just because that code has lots of dependencies to the other stuff in that method, but I did create an overload that doesn't have the flag in its signature. We were also able to remove the weird flag for verifying the offsets from the `assertContinuumMessagesAsExpected()`, which I think is the weirdness you were getting at.